pkgs/development/python-modules/pyreadline/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, fetchPypi, buildPythonPackage }:
+
+buildPythonPackage rec {
+  pname = "pyreadline";
+  version = "2.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    extension = "zip";
+    hash = "sha256-RTBZL8LoWyWxqfeWZEM9oJI3waJw5NeOpao6LHIp4tE=";
+  };
+
+  # https://github.com/pyreadline/pyreadline/issues/74
+  doCheck = false;
+
```

Comment:
Missing pythonImportsCheck or is that also broken?

Reply:
Looks like pyreadline is not needed at all, I removed it

https://github.com/iamtalhaasghar/yewtube/pull/105